### DISCUSSION TITLE
Missing filament profile Bambu basic PETG

### DISCUSSION THREAD

**Discussion by alldaywarrior** [OrcaSlicer/OrcaSlicer#1134] [Q&A] [answered]
It seems that the orcaslicer does not have bambu basic petg profile? I am a pretty noob at setting up the perfect settings for a filament type. Can orcaslicer please add this profile to their program? Bambuslicer itself has it too. Would be nice if someone can make it happen, thankyou :)

**Comment by fr3ak2402** [comment]
In which version was this added in Bambulab Studio?

**Reply by alldaywarrior** [reply]
I got version 1.6.2.4 as far as I know I never updated the program. This version has the bambu basic petg profile
![Uploading 16849260682837135065820946187619.jpg…]()

**Reply by fr3ak2402** [reply]
@alldaywarrior 

So the Bambulab PETG Basic is already included in the OrcaSlicer, here is a picture of the current files of the OrcaSlicer from BBL printer filaments.

<img width="1567" alt="Bildschirmfoto 2023-05-24 um 13 00 08" src="https://github.com/SoftFever/OrcaSlicer/assets/96239814/634507ca-f10b-4727-87de-be7cbe963e1d">

Which OrcaSlicer version are you using?

**Reply by alldaywarrior** [reply]
1.6.2?

**Reply by alldaywarrior** [reply]
I only got petg-cf

**Reply by fr3ak2402** [reply]
I'll take a look at the problem now, I need some information from you.

Which BBL printer do you have?

**Reply by alldaywarrior** [reply]
I got the p1p

**Reply by fr3ak2402** [reply]
okay. wait a moment

**Reply by fr3ak2402** [reply]
So the filaments are included in the OrcaSlicer V 1.6.3 beta. If you use the V1.6.2, then unfortunately still in there.

Here you can download version V1.6.3 beta.
https://github.com/SoftFever/OrcaSlicer/releases/tag/v1.6.3-beta

If you then go to Add filaments in the slicer under Filaments, you will also see the PETG Basic

**Reply by fr3ak2402** [reply]
If you need help, you can simply ask here

**Reply by alldaywarrior** [reply]
![16849268231616848172825427867889](https://github.com/SoftFever/OrcaSlicer/assets/27778908/dfe3b2a4-4038-4a82-863f-04fe9707a460)

**Reply by alldaywarrior** [reply]
Ah thanks I will look into it

**Reply by alldaywarrior** [reply]
![16849269979974879192183707323681](https://github.com/SoftFever/OrcaSlicer/assets/27778908/12afcf50-e1ba-41a7-9b87-32cb54c6a8c0)

Sadly it wasnt added for the p1p in version 1.6.3 neither

**Reply by alldaywarrior** [reply]
Well it is added when I open the beta version but weirdly enough it isnt in the files? Lol huh I dont understand anymore

**Reply by fr3ak2402** [reply, marked as answer]
I downloaded this zip from the link below
https://github.com/SoftFever/OrcaSlicer/releases/tag/v1.6.3-beta

<img width="1258" alt="Bildschirmfoto 2023-05-24 um 13 19 39" src="https://github.com/SoftFever/OrcaSlicer/assets/96239814/d6b506ee-1eb6-423e-b5cb-eef9314137ee">

Then go to the following folder /resources/profiles/BBL/filament in the zip

Then the filaments are there for me
<img width="748" alt="Bildschirmfoto 2023-05-24 um 13 19 10" src="https://github.com/SoftFever/OrcaSlicer/assets/96239814/a67e9990-1785-4262-9aa7-6537f1504b24">

**Reply by fr3ak2402** [reply]
Are the filaments now also in your slicer under the P1P?

**Reply by alldaywarrior** [reply]
I got it. I transfered all profiles from version 1.6.3 to 1.6.2 thankyou :)
